packages/grafana-ui/src/components/Forms/Checkbox.tsx

```diff
@@ -32,6 +32,7 @@ export const getCheckboxStyles = stylesFactory((theme: GrafanaTheme) => {
       position: relative;
       padding-left: ${checkboxSize};
       vertical-align: middle;
+      line-height: 1.2;
```

Comment:
not sure about this, might be better to set line-height to formInputHeight like do with other inputs

Reply:
I ran out of ideas to align the checkbox with its label. The checkbox indicator having absolute positioning makes it difficult.

Reply:
I moved this style to the specific checkbox in ActionRow.